config/settings.yml

```diff
@@ -1,7 +1,7 @@
 feature_flags:
   dev_auth:
     local: <%= ENV.fetch("DEV_AUTH_ENABLED", false) %>
-    development: <%= ENV.fetch("DEV_AUTH_ENABLED", false) %>
+    development: <%= ENV.fetch("DEV_AUTH_ENABLED", true) %>
```

Comment:
so this makes it impossible to be set to false... as if you set it to false it will return a string which is truthy... :) 

we are better off setting the ENV var if possible. :)